release_images.yml

```
@@ -74,9 +74,9 @@ release_images:
     version: "1.15.5"
     training:
       device_types: ["gpu"]
-      python_versions: ["py37"]
+      python_versions: ["py36"]
```

We need to release the following images:
Py36 - Training CPU/GPU-CUDA-10.0
Py37 - Training CPU/GPU-CUDA-10.0

These can be configured by making the following changes:
```suggestion
      device_types: ["cpu", "gpu"]  
      python_versions: ["py36", "py37"]
      os_version: "ubuntu18.04
      cuda_version: "cu100"
      example: False
      disable_sm_tag: False
      force_release: False
```

---

we have released py37 training images i think.
Here is the tag from this morning: https://github.com/aws/deep-learning-containers/releases/tag/v8.12-tf-1.15.5-tr-gpu-py37